docs/data/sdks/ampli-overview.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Ampli
+description: Overview to Amplitude Analytics SDK architecture and common usage
```

Comment:
Probably also need to change this to include some key words like "Ampli overview". It is used by the search bar.